### PR TITLE
LVRUBY-60: Add a share button on News page

### PR DESCRIPTION
**Pull request by kmandziy**
## JIRA ticket

https://ssu-jira.softserveinc.com/browse/LVRUBY-60

## Code reviewers

- [ ] @okohuch 
- [ ] @nataliakozoriz 

## Summary of issue

Add button in order to share link of current News

## Summary of change

Added two links for sharing News via Twitter and Facebook

## Review checklist

- [ ] Rubocop doesn't raise any warnings and offenses
- [ ] New features are covered with new specs
- [ ] All specs pass
Screenshot from 2019-05-07 14-17-31

### CODE REVIEW

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.6%** covered at **2.74** hits/line

File: app/views/news/index.html.haml [hunk 1]
```diff
@@ -24,8 +24,10 @@
             %h6.m-0.text-dark
               = link_to t('news.links.read_more'), news_path(news)
               = fa_icon 'eye', text: '1233', class: 'ml-4'
-              = fa_icon 'share', text: t('news.links.share'), class: 'ml-4'
               = fa_icon 'tags', text: t('news.links.category'), class: 'ml-4'
+              = fa_icon 'share', class: 'ml-4', 'data-target' => '#exampleModal', |
```
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/views/news/index.html.haml]: Removed " | "

File: app/views/news/_modal_share_window.html.haml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#exampleModal.modal.fade{ 'aria-hidden' => 'true', |
```
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/views/news/_modal_share_window.html.haml]: Removed " | "

File: app/views/news/_modal_share_window.html.haml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+'aria-labelledby' => 'exampleModalLabel', role: 'dialog', tabindex: '-1' }
+  .modal-dialog{ :role => 'document' }
+    .modal-content.p-5
+      %button.text-right.close{ 'aria-label' => 'Close', |
```
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/views/news/_modal_share_window.html.haml]: Removed " | "

File: app/views/news/_modal_share_window.html.haml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+      'data-dismiss' => 'modal', type: 'button' }
+        %span{ 'aria-hidden' => 'true' }
+          = fa_icon 'times'
+      .modal-header
+        %h5#exampleModalLabel.modal-title
+          = t('news.links.share')
+        %h5
+          = link_to fa_icon('facebook-square'), |
```
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/views/news/_modal_share_window.html.haml]: Removed " | "

File: app/views/news/_modal_share_window.html.haml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+            'https://www.facebook.com/sharer/sharer.php?' |
+            "u=#{news_url(news)}&quote=#{news.title}", |
+            class: 'social-icon'
+          = link_to fa_icon('twitter-square'), |
```
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/views/news/_modal_share_window.html.haml]: Removed " | "

File: app/views/news/_modal_share_window.html.haml [hunk 6]
```diff
@@ -0,0 +1,20 @@
+#exampleModal.modal.fade{ 'aria-hidden' => 'true', |
+'aria-labelledby' => 'exampleModalLabel', role: 'dialog', tabindex: '-1' }
+  .modal-dialog{ :role => 'document' }
+    .modal-content.p-5
+      %button.text-right.close{ 'aria-label' => 'Close', |
+      'data-dismiss' => 'modal', type: 'button' }
+        %span{ 'aria-hidden' => 'true' }
+          = fa_icon 'times'
+      .modal-header
+        %h5#exampleModalLabel.modal-title
```
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/views/news/_modal_share_window.html.haml]: Removed " | "

[the remark this answers is not in the document]
Comment (kmandziy) [reply app/views/news/_modal_share_window.html.haml]: Changed place of id

File: app/views/news/_modal_share_window.html.haml (hunk 3 above)
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/views/news/_modal_share_window.html.haml]: Removed " | "

File: app/views/news/_modal_share_window.html.haml [hunk 7]
```diff
@@ -0,0 +1,18 @@
+.modal.fade#exampleModal{ 'aria-hidden' => 'true',
+'aria-labelledby' => 'exampleModalLabel', role: 'dialog', tabindex: '-1' }
+  .modal-dialog{ :role => 'document' }
+    .modal-content.p-5
+      %button.text-right.close{ 'aria-label' => 'Close',
+      'data-dismiss' => 'modal', type: 'button' }
```
Comment (okohuch) [inline app/views/news/_modal_share_window.html.haml]: change hash-rocket syntax to the new one

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.6%** covered at **2.74** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.6%** covered at **2.74** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.6%** covered at **2.74** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.6%** covered at **2.74** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.6%** covered at **2.74** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.6%** covered at **2.74** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Pull request merged**